ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-release-1.0.yaml

```diff
@@ -97,7 +97,6 @@ tests:
   commands: ci/prow/check-commits-count
   container:
     from: src
-  optional: true
```

Comment:
@qbarrand @mresvanis Why did we make that job optional to begin with?
I am seeing https://github.com/rh-ecosystem-edge/kernel-module-management/pull/401#issuecomment-1403608432 but no description of the issue and the job history seems green.

Reply:
We did because it fails when we want to merge commits into a branch that is not `main`, such as `release-1.0`. We should leave it optional until we fix it.

Reply:
Fixed.